### PR TITLE
Remove crossplane

### PR DESCRIPTION
**Pull request by avnes**
- **Remove crossplane**
- **Update pre-commit**

## Describe your changes
Indirectly related to https://github.com/dfds/cloudplatform/issues/2983 because some cleanup will make that issue easier to handle.

## Checklist before requesting a review
- [x] I have tested changes in my sandbox
- [x] I have added the needed changes in the `test/integration` folder to apply my changes in QA. [Read the guide on adding environment variables in QA](https://wiki.dfds.cloud/en/ce-private/atlantis/adding-env-vars)
- [x] I have rebased the code to master (or merged in the latest from master)

## Is it a new release?
- [x] Apply a release tag `release:(major|minor|patch)`, following semantic versioning in [this guide](https://semver.org/) or `norelease` if there is no changes to the Terraform code

### CODE REVIEW

File: compute/k8s-services/vars.tf [hunk 1]
```diff
@@ -1273,7 +1109,7 @@ variable "kafka_exporter_deploy" {
 }
 
 variable "kafka_exporter_clusters" {
-  type        = map
+  type        = map(any)
```
Comment (SEQUOIIA) [inline compute/k8s-services/vars.tf:1112]: Seems unrelated. Any reason or just cause why not?

**Review by SEQUOIIA — DISMISSED** [review]
Left a tiny comment, no biggie.

File: compute/k8s-services/vars.tf (hunk 1 above)
Comment (avnes) [reply compute/k8s-services/vars.tf:1112]: Format and linting on save....
